portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase

```diff
@@ -38,12 +38,61 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8143 | Test Summary: Owner can delete users"
-	@ignore = "true"
 	@priority = "5"
 	test ViewOwnerCanDeleteUser {
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUserManagement();
+
+		ACSettings.addUser(userInvite = "user@liferay.com");
+
+		ACUtils.searchBar(searchItem = "user");
+
+		ACSettings.assertUserListEmail(userEmail = "user@liferay.com");
+
+		ACSettings.deleteUser(userDelete = "user@liferay.com");
+
+		AssertTextNotPresent(
+			locator1 = "ACSettings#USER_LIST_EMAIL",
+			value1 = "user@liferay.com");
+
+		ACWorkspace.signOut();
+
+		ACLogin.loginAs(
+			emailAddress = "corbin.murakami@faro.io",
+			password = "test");
+
+		ACWorkspace.openWorkspace();
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUserManagement();
+
+		AssertElementNotPresent(locator1 = "ACSettings#DELETE_USER");
```

Comment:
Hi @joshitagaki , I'm not sure what this assert is checking. Do you think we should assert a specific deleted user (in this case user@liferay.com) is not present?

Reply:
Thanks Calvin! Actually the path isn't checking a specific deleted user but if there are any places that you can delete a user. So yea the wording is a bit off but I want to check that a non-admin isn't able to delete any users.

Reply:
ahh thank you for clarifying